### DISCUSSION TITLE
Early tester for your new framework?

### DISCUSSION THREAD

**Discussion by issacclee** [sceneview/sceneform-android#24] [Ideas]
Hi Thomas,
This might be an odd place to ask this question, as you mentioned in this post(https://github.com/google/filament/issues/3669#issuecomment-803053527), that you are working on a RealityKit like framework based on Filament and ArCore. I was wondering if you need some tester for your new framework, we have some experienced arcore/sceneform programmers and huge amount of ready to use glb assets with various extensions encoded. We would very much like to help, cause we love your work on sceneform fork. Thanks!

**Comment by ThomasGorisse** [comment]
Thanks. It will be pleasure to share with you a beta testable version.
I'll let you know when it's pushed but I still have some work on it to make it ready to use
...and Scenform is still taking me a little time to be stable until the new framework comes.

**Reply by issacclee** [reply]
Cool! Appreciate that! 👍

**Comment by ThomasGorisse** [comment]
I started pushing the new ARCore/Filament SDK here : [RealityCore](https://github.com/ThomasGorisse/realitycore)

Mostly everything is ready and tested but I prefer checking and being sure everything is clean and well documented instead of pushing everything at one time. I'll push the SDK files by files next coming days if you want to have a look.
